chunker_test.go

```diff
@@ -185,7 +185,7 @@ func TestChunkerWithRandomPolynomial(t *testing.T) {
 	// generate a new random polynomial
 	start := time.Now()
 	p, err := chunker.RandomPolynomial()
-	OK(t, err)
+	assert.Nil(t, err)
```

Comment:
Hm, the documentation of the assert library sounds like it does not terminate the test, which the old `OK()` function did. I think here you should use the `require.Nil()` instead of `assert.Nil()`. Is that correct?